packages/core/admin/server/src/controllers/__tests__/api-token.test.ts

```diff
@@ -2,7 +2,7 @@ import { errors } from '@strapi/utils';
 import { omit } from 'lodash/fp';
 // @ts-expect-error - types are not generated for this file
 // eslint-disable-next-line import/no-relative-packages
-import createContext from '../../../../../../../test/helpers/create-context';
+import createContext from '../../../../../../../tests/scripts/helpers/create-context';
```

Comment:
is this really a script ?

Reply:
Ahh, true, we might want to flatten that, since `setup` isn't really script either